UPGRADING.md

```diff
@@ -0,0 +1,105 @@
+# Upgrading Guide
+
+See the [CHANGELOG.md](./CHANGELOG.md) for detailed information about what has changed between versions.
+
+This guide is useful to figure out what you need to do between breaking changes.
+
+As always, [submit issues](https://github.com/FortAwesome/vue-fontawesome/issues/new) that you run into with this guide or with these upgrades to us.
+
+## 0.0.x to 0.1.0
+
+### Renamed packages
+
+The following packages have been renamed as part of 5.1.0 of Font Awesome.
+
+_All packages are in the [@fortawesome NPM scope](https://www.npmjs.com/search?q=scope:fortawesome&page=1&ranking=optimal)_
+
+| Old package(1)           | New package            |
+|--------------------------|------------------------|
+| fontawesome              | fontawesome-svg-core   |
+| fontawesome-free-solid   | free-solid-svg-icons   |
+| fontawesome-free-regular | free-regular-svg-icons |
+| fontawesome-free-brands  | free-brands-svg-icons  |
+| fontawesome-pro-solid    | pro-solid-svg-icons    |
+| fontawesome-pro-regular  | pro-regular-svg-icons  |
+| fontawesome-pro-light    | pro-light-svg-icons    |
+
+(1) Old packages have now been deprecated. They are still available but will only receive high priority patch release fixes.
+
+### No more default imports
+
+Recently we spent a good deal of time supporting TypeScript to enable us to
+create the Angular Font Awesome component. During that adventure we
+[were](https://basarat.gitbooks.io/typescript/docs/tips/defaultIsBad.html)
+[convinced](https://blog.neufund.org/why-we-have-banned-default-exports-and-you-should-do-the-same-d51fdc2cf2ad)
+that we were going to remove default exports from all of our components,
+libraries, and packages. This is complete with the umbrella release of `5.1.0` of Font Awesome.
+
+What does that mean?
+
+~~Old way:~~
+
+```javascript
+import fontawesome from '@fontawesome/fontawesome'
+import solid from '@fortawesome/fontawesome-free-solid'
+import faTwitter from '@fortawesome/fontawesome-free-brands/faTwitter'
+import FontAwesomeIcon from '@fortawesome/vue-fontawesome'
+
+import fontaweome.library.add(solid, faTwitter)
+```
+
+New way:
+
+```javascript
+import { library } from '@fontawesome/fontawesome-svg-core'
+import { fas } from '@fortawesome/fontawesome-free-solid'
+import { faTwitter } from '@fortawesome/free-brands-svg-icons'
```

Comment:
I notice that, here, you don't suggest a _named submodule import_, though you used a _default submodule import_ above. 

In 5.1, you could either do a named import on the top-level icon package, or a named import on the submodule. It might be worth mentioning both of those options, because I think the named submodule import is an option that avoids the concern about tree-shaking the whole icon package.

Reply:
To be more specific, this also works:
```javascript
import { faTwitter } from '@fortawesome/free-brands-svg-icons/faTwitter'
```

Reply:
Gotcha. I've updated. Thanks @mlwilkerson